src/script/view_model/ContentViewModel.ts

```diff
@@ -228,19 +298,10 @@ export class ContentViewModel {
       });
   }
 
-  /**
-   * Opens the specified conversation.
-   *
-   * @note If the conversation_et is not defined, it will open the incoming connection requests instead
-   *
-   * @param {Conversation|string} conversation Conversation entity or conversation ID
-   * @param {Object} options State to open conversation in
-   * @param {Message} [options.exposeMessage] Scroll to message and highlight it
-   * @param {boolean} [options.openFirstSelfMention=false] Open first self mention instead of passed message
-   * @param {boolean} [options.openNotificationSettings=false] Open notification settings of conversation
-   * @returns {undefined} No return value
-   */
-  showConversation = (conversation, options = {}) => {
+  showConversation = (
```

Comment:
Please re-add the JSDoc without the type declarations. Example:

```js
/**
 * ...
 * @param conversation Conversation entity or conversation ID
 * ...
 */

Reply:
https://github.com/wireapp/wire-webapp/pull/9173/commits/4427041e2014c1861a0b8181a295ff1ffba0d50a